src/dom/dom.js

```diff
@@ -2221,12 +2221,23 @@ p5.prototype.createCapture = function(...args) {
   let useAudio = true;
   let constraints;
   let callback;
+  let flipped = false;
+
   for (const arg of args) {
     if (arg === p5.prototype.VIDEO) useAudio = false;
     else if (arg === p5.prototype.AUDIO) useVideo = false;
-    else if (typeof arg === 'object') constraints = arg;
-    else if (typeof arg === 'function') callback = arg;
+    else if (typeof arg === 'object') {
+      constraints = arg;
+      flipped = constraints.flipped || false;
+    }
+    else if (typeof arg === 'boolean') {
```

Comment:
I think it's more futureproof to force people to use the settings object (`createCapture(VIDEO, { flipped: true })`) than to add additional overloads with a boolean argument (`createCapture(VIDEO, true)`) in case we need to add any other additional boolean settings. How do you feel about taking out this branch?